core_integrations/slow_pwm/icon.txt

```diff
@@ -0,0 +1 @@
+mdi:pwm
```

Comment:
This icon does not exists and thus cannot be used. This PR is currently invalid.